Website/Packages/datax-pipeline/src/modules/flowDefinition/flowActions.js

```diff
@@ -131,6 +145,13 @@ export const updateDatabricksToken = databricksToken => dispatch => {
     });
 };
 
+export const updateIsDatabricksSparkType = isDatabricksSparkType => dispatch => {
```

Comment:
Ok. you already created this but looks like you don't use this ?

Reply:
removed it. since its not being used.. cant use in init flow since this method doesnot take dispatch as a parameter